extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/runtime/OidcClientImpl.java

```diff
@@ -240,6 +240,7 @@ private UniOnItem<HttpResponse<Buffer>> postRequest(
                 body.add(entry.getKey(), entry.getValue());
             }
         }
+        LOG.debugf("Token endpoint: %s, request params: %s, headers: %s", request.uri(), body, request.headers());
```

Comment:
It's not obvious that we won't pay a price for `uri` and `headers` calls so I would recommend to test `isDebugEnabled()` in this case.

Reply:
Sure, I added the check to the `quarkus-oidc` client code too